.changeset/seven-dots-serve.md

```diff
@@ -0,0 +1,34 @@
+---
+'@backstage/plugin-catalog': minor
+---
+
+Exported `CatalogTable.defaultColumnsFunc` to create a seam for defining the columns in `<CatalogTable />` of some Kinds while using the default columns for the others.
+This is useful for defining the columns of a custom Kind or to redefine the columns for a built-in Kind.
+
+```diff
```

Comment:
I think that this diff would be better placed in documentation instead of the changelog.

Maybe moving some of this into https://backstage.io/docs/features/software-catalog/catalog-customization instead and droppping it from the changeset?

Reply:
Just my two cents: I agree but that page is not good, was running through it over the holidays and a lot of it is broken (haven't had time to fix in a PR). Feel like this would fit in nicely with this: https://github.com/backstage/backstage/issues/19376

Reply:
I agree about removing the diff too, assuming it will exist in documentation somewhere (which is what you are suggesting @benjdlambert). A reason I went with the changeset approach is because I often find it difficult to determine how to use a new feature/seam when going through the release notes.

I think @awanlin brings up a good point about that page though. So, is it worthwhile to add guidance into the current https://backstage.io/docs/features/software-catalog/catalog-customization or should we wait until work on #19376 begins?

I think the wording of the changeset is fine even without the diff, do you both agree?
> Exported `CatalogTable.defaultColumnsFunc` to create a seam for defining the columns in `<CatalogTable />` of some Kinds while using the default columns for the others.
This is useful for defining the columns of a custom Kind or to redefine the columns for a built-in Kind.

Reply:
@GoFightNguyen can probably make it a little simpler:

```
Exported `CatalogTable.defaultColumnsFunc` for defining the columns in `<CatalogTable />` of some Kinds while using the default columns for the others. This is useful for defining the columns of a custom Kind or to redefine the columns for a built-in Kind.
```

Reply:
Thanks for the suggestion @benjdlambert; the changeset has been simplified